integration/docker-compose.spdk.yml

```diff
@@ -16,19 +16,20 @@ services:
       opi:
         ipv4_address: 10.127.127.4
     command: |
-      sh -c 'sync; echo 1 | sudo tee /proc/sys/vm/drop_caches  && \
+      sh -x -c 'sync; echo 1 > /proc/sys/vm/drop_caches  && \
```

Comment:
Do we want to use `sh` or `bash` here? I will approve as-is, but curious why you picked `sh` instead of `bash`.

Reply:
no reason